### ISSUE TITLE
APT package source definition specifies origin as "." instead of "grafana"

### ISSUE THREAD

**Issue by vbackeberg** [grafana/grafana#86693]
### What happened?

When creating a package source file for the Grafana APT repository as described in [the installation guide](https://grafana.com/docs/grafana/latest/setup-grafana/installation/debian/#install-from-apt-repository), it creates a package source file containing the following lines:

```
Origin: . stable
[...]
```

### What did you expect to happen?

The package source file should contain a proper origin identifier such as:

```
Origin: grafana stable
[...]
```

### Did this work before?

Not to my knowledge

### How do we reproduce it?

1. Perform the steps described in: https://grafana.com/docs/grafana/latest/setup-grafana/installation/debian/#install-from-apt-repository
2. `sudo nano /var/lib/apt/lists/apt.grafana.com_dists_stable_InRelease`

### Is the bug inside a dashboard panel?

no

### Environment (with versions)?

Grafana: 10.4.2 arm64
OS: Ubuntu 22.04.4 LTS

### Grafana platform?

A package manager (APT, YUM, BREW, etc.)

**Comment by evgeni** [comment]
cc @RichiH @bastischubert 

Turns out this was already reported :)

(context: https://chaos.social/@zhenech/112461158047104144)

**Comment by ChrLau** [comment]
Hi,

would be nice if this could be fixed in time as this actually allows no proper separation between repositories when using unattended-upgrades.
While it does work with the following line:
```
        //Grafana
        "origin=. stable,n=stable,a=stable,l=. stable";
```

Something like this would be better:
```
        //Grafana
        "origin=grafana,n=stable,a=stable,l=grafana-stable";
```

**Comment by evgeni** [comment]
```
 500 https://apt.grafana.com stable/main amd64 Packages
     release o=. stable,a=stable,n=stable,l=. stable,c=main,b=amd64
     origin apt.grafana.com
```

**Comment by Lukas-Heindl** [comment]
Just stumbled over this as well.

Yes I know https://wiki.debian.org/DebianRepository/Format#Origin says the `origin` field is optional. Still `grep -i grafana /var/lib/apt/lists/apt.grafana.com_dists_stable_InRelease` reveals there is not even a tiny thing indicating this is a grafana repository in the release file which makes it literally impossible to come up with a `unattended upgrades` rule for the package leading to people missing out on updates potentially running old (unsupported) versions of grafana.

**Comment by evgeni** [comment]
We're using `site=apt.grafana.com,component=main` for unattended upgrades, but that requires you to use the primary site and not a local mirror.

**Comment by Lukas-Heindl** [comment]
Oh nice this seems to be working, thanks.

The `site` parameter is something I didn't find in apt docs so far neither does it occur in many tutorials/blog posts regarding unattended upgrades (e.g. https://benheater.com/configuring-unattended-upgrades-on-debian/). But still it seems to work 🤷

Regarding Grafana: Still I think it would be nice to fill these typical fields like `origin` to make it more "standard". Nonetheless, with the "fix" for unattended-upgrades this might be not so urgent anymore (in my opinion)

**Comment by ChrLau** [comment]
Was surprised to read about that `site` parameter too. But it's indeed listed in the `/etc/apt/apt.conf.d/50unattended-upgrades` file in Debian Bookworm and Trixie (have no older systems to check).
It's also mentioned in the Readme from upstream: https://github.com/mvo5/unattended-upgrades

**Comment by evgeni** [comment]
Yeah, I think unattended upgrades docs is where I have it from, after being annoyed by exactly this repo.

**Comment by ronanchilvers** [comment]
It would be nice if this was fixed properly. Commenting to try to keep the issue alive.

**Comment by thulle** [comment]
Another vote on getting this fixed, especially as the site= workaround doesn't work on ubuntu.
